Dockerfile

```diff
@@ -23,6 +23,8 @@ ENV ROCKET_LIMITS "{json=10485760}"
 # syntect_server does not need a secret key since it uses no cookies, but
 # without one set Rocket emits a warning.
 ENV ROCKET_SECRET_KEY "SeerutKeyIsI7releuantAndknvsuZPluaseIgnorYA="
+# See issue in Rocket for why we need this: https://github.com/SergioBenitez/Rocket/issues/928
```

Comment:
```suggestion
# When keep-alive is on, we observe connection resets in our Go clients of
# syntect_server. It is unclear why this is, especially because our Go clients do
# not reuse the connection (i.e. we make a fresh connection every time).
# Disabling keep-alive does resolve the issue though, our best guess is that
# this is a bug in Hyper 0.10 (see https://github.com/SergioBenitez/Rocket/issues/928).
```